packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts

```diff
@@ -51,7 +51,7 @@ describe('build-schema', () => {
         expect(stringTypeToWrapper('date')).to.eql(Date);
       });
 
-      it('returns Object for "object"', () => {
+      it('returns  for "object"', () => {
```

Comment:
Can we keep this as is to be consistent with the other tests?

Reply:
Yeah, I accidentally had it removed during find/replace for other occurrences.